### ISSUE TITLE
Failing to build on macOS nightly

### ISSUE THREAD

**Issue by michalhosna** [PoiScript/strong-xml#23]
When trying to compile on macOS using nightly compile compilation fails with `cannot perform const operation using T` for crate `jetscii`. 

Upgrading to `jetscii: "0.5"` solves this problem. See: https://github.com/shepmaster/jetscii/commit/e35a6c759f6ade55df2a8afdddc347e77209a3a6

Compiler error:
```
 Compiling jetscii v0.4.4
error: generic parameters may not be used in const operations
   --> /Users/michalhosna/.cargo/registry/src/github.com-1ecc6299db9ec823/jetscii-0.4.4/src/simd.rs:109:13
    |
109 |             T::CONTROL_BYTE,
    |             ^^^^^^^^^^^^^^^ cannot perform const operation using `T`
    |
    = note: type parameters may not be used in const expressions
    = help: use `#![feature(const_generics)]` and `#![feature(const_evaluatable_checked)]` to allow generic const expressions

error: generic parameters may not be used in const operations
   --> /Users/michalhosna/.cargo/registry/src/github.com-1ecc6299db9ec823/jetscii-0.4.4/src/simd.rs:148:13
    |
148 |             T::CONTROL_BYTE,
    |             ^^^^^^^^^^^^^^^ cannot perform const operation using `T`
    |
    = note: type parameters may not be used in const expressions
    = help: use `#![feature(const_generics)]` and `#![feature(const_evaluatable_checked)]` to allow generic const expressions

error: aborting due to 2 previous errors

error: could not compile `jetscii`
```

**Comment by PoiScript** [comment]
sorry for the inconvenience, i just published v0.6.3 which upgraded jetscii to 0.5. now it should work.

**Comment by michalhosna** [comment]
Thanks! No need to apologise! Keep up the great work.
